dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -12,4 +12,50 @@ def index
   def show
     render json: @section.summarize
   end
+
+  # POST /api/v1/sections
+  # Create a new section
+  def create
+    # Validate login_type
+    # TODO: Push validation into model?
+    login_type = (['none', '', nil].include? params[:login_type]) ? 'email' : params[:login_type]
```

Comment:
In the pegasus API, `'none'` was the only valid way to indicate no login type.  I was hoping to normalize that a little without removing support for the old way of doing it.

Now that I think about it though, maybe it shouldn't be valid to create a section without a login type at all?  Especially now that we're keying single-table-inheritance on it. We can depend on the UI to provide defaults here, and reject requests that don't include it.

@joshlory thoughts?